platform/solr/solr-xpath/pom.xml

```diff
@@ -104,5 +104,10 @@
             <artifactId>lux</artifactId>
             <version>${lux.version}</version>
         </dependency>
+        <dependency>
+            <groupId>net.sf.saxon</groupId>
+            <artifactId>Saxon-HE</artifactId>
+            <version>9.5.1-3</version>
```

Comment:
We would need to update the version of Saxon used by the Lux fork.  This might also impact Solr index compatibility which would block it from going into a bug fix release.